roles/rsyslog/defaults/main.yml

```diff
@@ -359,15 +301,11 @@ __rsyslog_conf_common_defaults:
 
       - comment: 'Set default permissions for all log files'
         options: |-
-          $FileOwner {{ rsyslog_file_owner }}
-          $FileGroup {{ rsyslog_file_group }}
+          $FileOwner root
+          $FileGroup root
```

Comment:
Are these parameters required?  If not, I think they can be omitted.

Reply:
Nope.  The default is root.  Removing them.  Thanks!
```
./tools/omfile.c:	{ "fileowner", eCmdHdlrUID, 0 },
./plugins/imptcp/imptcp.c:	{ "fileowner", eCmdHdlrUID, 0 },
./tools/omfile.c:	{ "filegroup", eCmdHdlrGID, 0 },
./plugins/imptcp/imptcp.c:	{ "filegroup", eCmdHdlrGID, 0 },
```

Reply:
Besides, they are all obsolete legacy directives for omfile...

`$FileOwner` - `process user`, by default
`$FileGroup` - `process user's primary group`, by default
`$FileCreateMode` - `0644`, by default
`$DirCreateMode` - 0700, by default
`$Umask` - rsyslogd's process umask; if in doubt, use `$umask 0000`

I'd think it's guaranteed the process user is root with the linux system role deployment.  And we want to set the default permission 0700 to /etc/rsyslog.d, don't we?  Lastly, can we expect root's umask is 0000, by default?  If all is clear, I'm going to get rid of this entire 'Set default permissions for all log files'.

Reply:
> And we want to set the default permission 0700 to /etc/rsyslog.d, don't we? 

Yes, if not set by rpm install

> Lastly, can we expect root's umask is 0000, by default?

I think we can assume umask is set appropriately, yes.

> If all is clear, I'm going to get rid of this entire 'Set default permissions for all log files'.

ok

Reply:
Thanks, @richm.  I've removed the `Set default permissions for all log file`s rule.
Just as a record, here are the config file ownerships and modes.  By default, `/etc/bashrc` sets `umask 022` for the privileged users (but in this case, it did not matter).
```
Note: without ownership and mode changes, the following settings are made.
-r--------. 1 root root 134 Mar 15 15:48 rsyslog.conf
drwxr-xr-x. 2 root root 277 Mar 15 15:48 rsyslog.d
-r--------. 1 root root 118 Mar 15 15:48 rsyslog.d/##-filename.conf
```

Reply:
hmm - not sure why rsyslog.d is 0755 - but I don't think that should concern us - if anything, it should be changed in the rpm, but I don't think it is a problem

Reply:
Ok, the mode 0755 for /etc/rsyslog.d is coming from the spec file like this.  
```
install -d -m 755 %{buildroot}%{_sysconfdir}/sysconfig
install -d -m 755 %{buildroot}%{_sysconfdir}/logrotate.d
install -d -m 755 %{buildroot}%{_sysconfdir}/rsyslog.d
install -d -m 700 %{buildroot}%{rsyslog_statedir}
install -d -m 700 %{buildroot}%{rsyslog_pkidir}
install -d -m 755 %{buildroot}%{rsyslog_docdir}/html
```
But I also found the logging role is setting more ownerships and modes which looks unnecessary.  https://github.com/linux-system-roles/logging/pull/95/commits
Could you please review this second commit, too?  Thanks!